docs/core/scene.md

```diff
@@ -52,7 +52,7 @@ The scene inherits from the [`Entity`][entity] class so it inherits all of its p
 | enter-vr     | User has entered VR and headset started presenting content. |
 | exit-vr      | User has exited VR and headset stopped presenting content.  |
 | loaded       | All nodes have loaded.                             |
-| render-start | Render loop has started.            |
+| renderStarted| Render loop has started.            |
```

Comment:
Oh I see. Good eye. I will add my change as a property and update appropriately I guess. Not sure if you will all do this in one foul swoop later on, but just updating documentation when I find stuff.